x-pack/solutions/security/plugins/security_solution/common/api/detection_engine/alert_assignees/set_alert_assignees_route.gen.ts

```diff
@@ -15,20 +15,14 @@
  */
 
 import { z } from '@kbn/zod';
+import { isNonEmptyString } from '@kbn/zod-helpers';
 
 import { AlertIds } from '../../model/alert.gen';
-import { NonEmptyString } from '../../model/primitives.gen';
 
 export type AlertAssignees = z.infer<typeof AlertAssignees>;
 export const AlertAssignees = z.object({
-  /**
-   * A list of users ids to assign.
-   */
-  add: z.array(NonEmptyString),
-  /**
-   * A list of users ids to unassign.
-   */
-  remove: z.array(NonEmptyString),
```

Comment:
These descriptions have been removed. Is this expected?

Reply:
Yea, I replaced it with `z.array(z.string().min(1).superRefine(isNonEmptyString))`. Using `$ref` to reference `NonEmptyString` didn't allow me to overwrite the description for these params. Let me know if there is a better way to go about that.